H/hidapi/build_tarballs.jl

```diff
@@ -0,0 +1,61 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "hidapi"
+version = v"0.9.0"
+
+# Collection of sources required to complete build
+sources = [
+    "https://github.com/libusb/hidapi.git" => "7da5cc91fc0d2dbe4df4f08cd31f6ca1a262418f"
+]
+
+# Script template (structure is the same for all platforms, directory for make differs)
+# The additional eudev-dev package is to allow the ./configure step. For linux there are 
+# two backends: linux and libusb. This script builds only the 'libusb' backend
+# which doesn't needs libudev... 
+script = raw"""
+if [[ ${target} == *-linux-* ]]; then
+    apk add eudev-dev
```

Comment:
In this way you're installing a library for the native environment which is likely *not* what you want to do: https://juliapackaging.github.io/BinaryBuilder.jl/dev/FAQ/#Can-I-install-packages-in-the-build-environment?-1

Reply:
No, thats what I want and what I tried to explain in the comment above: Without this package the `./configure` step doesn't run through, because it requires `libudev` for _both_ backends (libusb and hidraw) but this script only builds the libusb related library, which doesn't need libudev. (=> compare `/libusb/hid.c`(libusb backend) with `/linux/hid.c` (linux backend).